### ISSUE TITLE
args not defined download.py callbacks

### ISSUE THREAD

**Issue by nkarpov** [tradel/appd-download-bot#1]
Awesome tool! I got it working but had some trouble - I had to comment out the args.quiet checks in the callbacks because they kept telling me args wasn't defined... what did I do wrong?

``````
nicks-mbp:scripts hellameta$ ./download-appdynamics -q -u email@appd.com -p password java-agent
Traceback (most recent call last):
  File "./download-appdynamics", line 10, in <module>
    main()
  File "/Library/Python/2.7/site-packages/appd/download/download.py", line 119, in main
    username=args.username, password=args.password, debug=args.debug)
  File "/Library/Python/2.7/site-packages/appd/download/zone.py", line 68, in __init__
    self.version = version
  File "/Library/Python/2.7/site-packages/appd/download/zone.py", line 131, in version
    avail_versions = self.get_versions()
  File "/Library/Python/2.7/site-packages/appd/download/zone.py", line 162, in get_versions
    Zone.status(self, 'Checking available product versions')
  File "/Library/Python/2.7/site-packages/appd/download/events.py", line 44, in fire
    handler(*args, **kargs)
  File "/Library/Python/2.7/site-packages/appd/download/download.py", line 10, in status_callback
    if not args.quiet:
NameError: global name 'args' is not defined```
``````

**Comment by tradel** [comment]
You're right. Worked on my machine and somehow passed the CI tests on Travis. Let me work on a fix and commit it. Thanks Nick.

**Comment by tradel** [comment]
Fixed in 76ca6739b84f828565e9b8e041b87a40a80971e6, Travis is testing now.

**Comment by tradel** [comment]
Builds passed in Travis, 0.1.6 uploaded to PyPI at https://pypi.python.org/pypi/AppDynamicsDownloader/0.1.6

@nkarpov you should be able to update by doing:

```
pip install --upgrade AppDynamicsDownloader
```
